app/src/main/java/com/example/mystore/ui/orderhistory/layouts/OrderRow.kt

```diff
@@ -31,13 +32,18 @@ fun OrderRow(orderModel: OrderModel) {
         Column {
             Text(
                 text = orderModel.items.size.toString() + (" " + if (orderModel.items.size > 1)
```

Comment:
Try to always use string interpolation instead of concatenation with `+`. You can also extract the if/else to a `val` above the `Text` so the string interpolation is achieved even more clearly.